### PR TITLE
feat: update chrome installation script

### PR DESCRIPTION
**Pull request by minikin**
- Resolves [#279](https://github.com/input-output-hk/catalyst-ci/issues/279)
- I decided to refrain from investing efforts to create a reusable solution for another browser due to other critical priorities.

### CODE REVIEW

File: earthly/flutter/Earthfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ VERSION 0.8
 
 IMPORT ./installer AS installer
 
-# flutter-src - save Flutter source code as artifact.
+# Save Flutter source code as artifact.
```
Comment (bkioshn) [inline earthly/flutter/Earthfile:5]: Earthly doc should begin with the target name
https://docs.earthly.dev/docs/earthly-command#earthly-doc

File: earthly/flutter/installer/Earthfile [hunk 2]
```diff
@@ -1,22 +1,48 @@
 VERSION 0.8
 
-# cspell: words Chromedriver Edgedriver edgedriver msedgedriver Geckodriver geckodriver
+# Installs the Chromium or Chrome driver.
+INSTALL_CHROMIUM:
+    FUNCTION
+
+    ARG --required PACKAGE_TYPE
+    ARG --required DOWNLOAD_URL
+
```
Comment (bkioshn) [inline earthly/flutter/installer/Earthfile:9]: I think you need this here too`LET BLUE='\033[0;34m'`

File: earthly/flutter/Earthfile [hunk 3]
```diff
@@ -17,19 +17,19 @@ flutter-src:
...
+    RUN apt-get update --fix-missing
+    RUN apt-get install -y apt-utils git curl gzip unzip bzip2 bash jq gpg lcov
 
     WORKDIR frontend
 
-    ARG chrome_version = 126.0.6478.61
-    ARG chrome_package_release = 1
-    ARG chromium_version = 126.0.6478.126-1~deb12u1
-    DO installer+INSTALL_CHROME_LINUX64 --chrome_version=$chrome_version --chrome_package_release=$chrome_package_release --chromium_version=$chromium_version
+    ARG TARGET_ARCH = amd64
```
Comment (bkioshn) [inline earthly/flutter/Earthfile:29]: Maybe use buildin args `TARGETARCH` 
https://docs.earthly.dev/docs/guides/multi-platform#making-use-of-builtin-platform-args

**Review by bkioshn — CHANGES_REQUESTED** [review]

File: earthly/flutter/Earthfile (hunk 1 above)
Comment (minikin) [reply earthly/flutter/Earthfile:5]: I read the docs. I wouldn't say I like how Earthly does it.

**Pull request merged**